### PR TITLE
Jakartafy Controllers

### PR DESCRIPTION
**Pull request by ivargrimstad**
Signed-off-by: Ivar Grimstad <ivar.grimstad@eclipse-foundation.org>

### CODE REVIEW

File: spec/src/main/asciidoc/chapters/controllers.asciidoc [hunk 1]
```diff
@@ -9,9 +9,9 @@ This chapter introduces the three components that comprise the
 Controllers
 ~~~~~~~~~~~
 
-[tck-testable tck-id-ctrl-method]#An _MVC controller_ is a JAX-RS [<<jaxrs20,5>>] resource method decorated by `@Controller`#.
+[tck-testable tck-id-ctrl-method]#Aa _Jakarta MVC controller_ is a Jakarta RESTful Web Services [<<jaxrs20,5>>] resource method decorated by `@Controller`#.
```
Comment (chkal) [inline spec/src/main/asciidoc/chapters/controllers.asciidoc:12]: `Aa` -> `A`?

File: spec/src/main/asciidoc/chapters/controllers.asciidoc [hunk 2]
```diff
@@ -177,7 +177,7 @@ public class MyController {
 
     @GET
     public String get() {
-        return "mybean.jsp"; // mybean.value accessed in JSP
+        return "mybean.jsp"; // mybean.value accessed in Jakarta Server Pages
```
Comment (chkal) [inline spec/src/main/asciidoc/chapters/controllers.asciidoc:180]: Maybe instead:

> mybean.value accessed in Jakarta Server Pages **page**

Or 

> mybean.value accessed in Jakarta Server Pages **file**


This also sounds weird. I guess the problem is that we often used the term "JSP" to refer to a JSP file and not to the spec itself.

File: spec/src/main/asciidoc/chapters/controllers.asciidoc [hunk 3]
```diff
@@ -286,10 +286,10 @@ Here is the JSP page for the hello-world example:
 </html>
 ----
 
-[tck-testable tck-id-jsp-el]#In a JSP, model properties are accessible via EL# [<<el30,6>>]. In the example above, the property `message` is read from the `greeting` model
+[tck-testable tck-id-jsp-el]#In a Jakarta Server Pages, model properties are accessible via EL# [<<el30,6>>]. In the example above, the property `message` is read from the `greeting` model
```
Comment (chkal) [inline spec/src/main/asciidoc/chapters/controllers.asciidoc:289]: Perhaps: 

> In a Jakarta Server Pages **file**.

Or simply 

> In Jakarta Server Pages    (remove "a")

File: spec/src/main/asciidoc/chapters/controllers.asciidoc [hunk 4]
```diff
@@ -304,14 +304,14 @@ Here is the corresponding example using Facelets instead of JSP:
 </html>
 ----
 
-[[mvc_uri]]
+[[MVC_uri]]
```
Comment (chkal) [inline spec/src/main/asciidoc/chapters/controllers.asciidoc:307]: Was this change intended?

File: spec/src/main/asciidoc/chapters/controllers.asciidoc [hunk 5]
```diff
@@ -304,14 +304,14 @@ Here is the corresponding example using Facelets instead of JSP:
...
 Building URIs in a View
 ^^^^^^^^^^^^^^^^^^^^^^^
 
 A typical application requires to build URIs for the view, which often refer to controller methods within the same application.
 Typical examples for such URIs include HTML links and form actions.
 As building URIs manually is difficult and duplicating path patterns between the controller class and the view is error prone,
-MVC provides a simple way to generate URIs using the `MvcContext` class.
+Jakarta MV provides a simple way to generate URIs using the `MvcContext` class.
```
Comment (chkal) [inline spec/src/main/asciidoc/chapters/controllers.asciidoc:314]: `MV` -> `MVC`

File: spec/src/main/asciidoc/chapters/controllers.asciidoc [hunk 6]
```diff
@@ -350,7 +350,7 @@ ${mvc.uri('BookController#detail', { 'isbn': 1234 })}
 [tck-testable tck-id-class-method-name]#The controller method is referenced using the simple name of the controller class and the corresponding method name separated by `pass:[#]`#.
 [tck-testable tck-id-param-map]#If the URI contains path, query or matrix parameters, concrete values can be supplied using a map#.
 Please note that the keys of this map must match the parameter name used in the `@PathParam`, `@QueryParam` or `@MatrixParam` annotation.
-[tck-testable tck-id-uri-encoding]#MVC implementations MUST apply the corresponding URI encoding rules depending on whether the value is used in a query, path or matrix parameter#.
+[tck-testable tck-id-uri-encoding]#Jakarta MV implementations MUST apply the corresponding URI encoding rules depending on whether the value is used in a query, path or matrix parameter#.
```
Comment (chkal) [inline spec/src/main/asciidoc/chapters/controllers.asciidoc:353]: `MV` -> `MVC`

File: spec/src/main/asciidoc/chapters/controllers.asciidoc (hunk 2 above)
Comment (ivargrimstad) [reply spec/src/main/asciidoc/chapters/controllers.asciidoc:180]: Maybe simply use Java Server Page when referring to a single file and Java Server Pages when referring to the technology?

File: spec/src/main/asciidoc/chapters/controllers.asciidoc (hunk 4 above)
Comment (ivargrimstad) [reply spec/src/main/asciidoc/chapters/controllers.asciidoc]: no, just sloppy work :)

File: spec/src/main/asciidoc/chapters/controllers.asciidoc (hunk 2 above)
Comment (chkal) [reply spec/src/main/asciidoc/chapters/controllers.asciidoc]: Yes, great idea!
